winml/api/Windows.AI.MachineLearning.idl

```diff
@@ -20,7 +20,7 @@ import "windows.storage.idl";
 
 namespace Windows.AI.MachineLearning 
 {
-    [contractversion(3)]
+    [contractversion(4)]
```

Comment:
when do we do API review? 💯

Reply:
we totes need to.  we have until march to have it finalized.  we should get it in front of damyan say by feb?

---
In reply to: [354042910](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354042910) [](ancestors = 354042910)